cmake/external/abseil-cpp.cmake

```diff
@@ -60,9 +58,34 @@ absl::node_hash_set
 absl::optional
 absl::raw_hash_set
 absl::raw_logging_internal
-absl::span
 absl::str_format
-absl::strings
+absl::str_format_internal
+absl::bits
+absl::fixed_array
+absl::numeric_representation
+absl::utility
+absl::type_traits
+absl::string_view
```

Comment:
This has caused a build problems for Power

```
CMake Error at onnxruntime_unittests.cmake:1756 (target_link_libraries):
  Target "test_execution_provider" links to:

    absl::string_view
```

and

```
/usr/bin/ld: cannot find -labsl::string_view
/usr/bin/ld: cannot find -labsl::nullability
/usr/bin/ld: cannot find -labsl::string_view
/usr/bin/ld: cannot find -labsl::nullability
```

Reply:
The library was added in [20230802.0](https://github.com/abseil/abseil-cpp/releases/tag/20230802.0). Could it because your abseil library is too old?  From the change https://github.com/abseil/abseil-cpp/commit/5668c20e027f161eeb0b0bfe6ddbc814705c1c6b, I don't see they explicitly disabled the library for PowePC.

Reply:
I tried updating from

    current version: 24.3.0
    latest version: 24.5.0

But got the same error

Reply:
Reverting `cmake/external/abseil-cpp.cmake` fixes the problem

Reply:
I just created a PR for this: #20962